app/server/utils/credit_transfer.py

```diff
@@ -46,6 +46,11 @@ def calculate_transfer_stats(total_time_series=False):
             .filter(CreditTransfer.transfer_type == TransferTypeEnum.PAYMENT).first().total
     )
 
+    total_exchanged = (
+        db.session.query(func.sum(CreditTransfer.transfer_amount).label('total'))
```

Comment:
so this would automatically do it just for the org right? do we have to worry about currency conversions? or would all exchanges be from/to the same currencies?

Reply:
i can definitely see opportunities for this to go wrong. Might be better to explicity filter on the token type being the organisation's so we don't have weirdness there. Even better still would be to list total exchanges in and total exchanges out as two different numbers, because that's where u get meaning from the data.

Reply:
Yes, this automatically filters based on the `?org=1` query string. 

@enjeyw Should we calculate _valid_ exchanges as ones with the `organisation.token_id` being equal to `exchange.from_token_id` OR `exchange.to_token_id` ?

Reply:
^ so based on that suggestion above, i'd split them out into two seperate ones -> exchanges into the currency and exchanges out - exchanges in and out combined isn't really that useful operational info IMO